fineract-provider/src/main/resources/sql/migrations/core_db/V362__Loan_Overpayment_Global_Configuration.sql

```diff
@@ -0,0 +1,23 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one
+-- or more contributor license agreements. See the NOTICE file
+-- distributed with this work for additional information
+-- regarding copyright ownership. The ASF licenses this file
+-- to you under the Apache License, Version 2.0 (the
+-- "License"); you may not use this file except in compliance
+-- with the License. You may obtain a copy of the License at
+--
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing,
+-- software distributed under the License is distributed on an
+-- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+-- KIND, either express or implied. See the License for the
+-- specific language governing permissions and limitations
+-- under the License.
+--
+
+--adding new configuration to enable/disable loan over payment
+
+INSERT INTO `c_configuration` (`name`, `enabled`,`description`)
+VALUES ('Enable-Loan-Overpayment',TRUE, 'configuration to enable loan payment beyond outstanding balance');
```

Comment:
As per my comments, shouldn't we have this as FALSE by default to retain the current behaviour as-is - unless configured otherwise